code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -558,28 +577,28 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 			drafted_rules[ruleset] = null
 			continue
 
-		if (check_blocking(ruleset.blocking_rules, rulesets_picked))
+		if (check_blocking(ruleset.blocking_rules, rulesets_picked, FALSE))
```

Comment:
use a named argument here for the third one

Reply:
Done